operations/fetch.go

```diff
@@ -189,75 +180,77 @@ func fetchSource(ac, rc *legacyClient, rootPath, taskId, token string, noPatch b
 }
 
 type cloneOptions struct {
-	token      string
-	owner      string
-	repository string
-	revision   string
-	rootDir    string
-	branch     string
-	depth      uint
+	repo     string
+	revision string
+	rootDir  string
+	branch   string
+	depth    uint
 }
 
 func clone(opts cloneOptions) error {
-	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
-	defer cancel()
-
-	var url string
-	if opts.token != "" {
-		url = fmt.Sprintf("https://%s@github.com/%s/%s.git", opts.token, opts.owner, opts.repository)
-	} else {
-		url = fmt.Sprintf("git@github.com:%v/%v.git", opts.owner, opts.repository)
-	}
-	cloneOpts := git.CloneOptions{
-		URL:      url,
-		Depth:    int(opts.depth),
-		Progress: os.Stdout,
+	// clone the repo first
+	cloneArgs := []string{"clone", opts.repo}
+	if opts.depth > 0 {
+		cloneArgs = append(cloneArgs, "--depth", fmt.Sprintf("%d", opts.depth))
 	}
 	if opts.branch != "" {
-		cloneOpts.ReferenceName = plumbing.ReferenceName(fmt.Sprintf("refs/heads/%s", opts.branch))
+		cloneArgs = append(cloneArgs, "-b", opts.branch)
 	}
-	repo, err := git.PlainCloneContext(ctx, opts.rootDir, false, &cloneOpts)
-	if err != nil {
-		return errors.Wrap(err, "error cloning repo")
-	}
-	workTree, err := repo.Worktree()
+
+	cloneArgs = append(cloneArgs, opts.rootDir)
+	grip.Debug(cloneArgs)
```

Comment:
If you're going to log with grip we should set a plain logger

Reply:
This is from the revert. Is this logging to the wrong place without setting the logger?

Reply:
No but it'll be printing out extra grip metadata (it was before too)